hack/testing/check-EFK-running.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
```

Comment:
I'll probably go with the approach of a way to make it verbose with a flag/env var.  I wouldn't expect someone to need to see the output by default (makes it slightly difficult to see the test results)